### DISCUSSION TITLE
allurectl does not see allurectl.yaml

### DISCUSSION THREAD

**Discussion by h8-rsd** [allure-framework/allure2#2466] [TestOps]
Hello everyone, I recently started using allure and ran into a problem. I need to send a report on the completion of auto-tests to Allure TestOps using allurectl. I created a file allurectl.yaml in which I entered the data, but for some reason allurectl does not see this file and requires me to specify the endpoint, token, etc. What should I do to make allure.yaml visible? I tried explicitly specifying the path to allure.yaml using --config; moved allure.yaml to the C:\Users\.allure folder;

This is what my allure.yaml file looks like:
`export ALLURE_ENDPOINT=https://allure.company.com
export ALLURE_TOKEN=55555555-5555-5555-5555-555555555555
export ALLURE_PROJECT_ID=100
export ALLURE_LAUNCH_NAME="Hello, world"`

**Comment by cheshi-mantu** [comment]
Hi @GTFOM8 
please refer to the docs of allurectl here: https://docs.qameta.io/allure-testops/ecosystem/allurectl/
wew described there all the options you might require when working with allurectl.

cheers
